README.md

```diff
@@ -50,6 +62,21 @@ If you are not running this client on Google Compute Engine, you need a Google D
   * If you want to use a new service account, click on **Create new Client ID** and select **Service account**. After the account is created, you will be prompted to download the JSON key file that the library uses to authorize your requests.
   * If you want to generate a new key for an existing service account, click on **Generate new JSON key** and download the JSON key file.
 
+``` js
+// Authorizing on a global basis.
+var projectId = process.env.GCLOUD_PROJECT_ID; // E.g. 'grape-spaceship-123'
+var keyFile = process.env.GCLOUD_KEYFILE; // E.g. /path/to/keyfile.json
+
+var gcloud = require('gcloud')({
+  projectId: projectId,
+  credentials: require(keyFile)
+});
```

Comment:
I would rather highlight using `keyFIlename` here, since we will do an async fetch and parse of the file.